src/components/Todo/todo.jsx

```diff
@@ -79,6 +94,14 @@ const Todo = () => {
     dispatch(editTodo(taskId, inputTask));
   };
 
+  const handlePaginationClick = (buttonText) => {
```

Comment:
see https://github.com/azo-wtag/todo-app-react/pull/4/files#r1172078490

```suggestion
  function handlePaginationClick(buttonText) {
```